File: Java/MathematicalPrograms/3DShapes/AreaAndPerimeterOfCube.java
import java.util.Scanner;
public class AreaAndPerimeterOfCube {
	public static void main(String[] args) {
		Scanner myObj = new Scanner(System.in);
		System.out.println("Enter the side of the cube");
		int side = myObj.nextInt();
		double tsa = 6*side*side;
		double lsa = 4*side*side;
		System.out.println("Total Surface Area of Cube is " + tsa);
		System.out.println("Lateral Surface Area of Cube is " + lsa);
	}
}

File: C++/MathematicalPrograms/3DShapes/AreaAndPerimeterOfCube.cpp
#include <iostream>
#include <string>
#include <math.h>

using namespace std;

int main() {
	int side;
	double tsa, lsa;
	cout << "Enter the side of the cube";
	cin >> side;
	tsa = 6*pow(side, 2);
	lsa = 4*pow(side, 2);
	cout << "Total Surface Area of Cube is " << tsa;
	cout << "\nLateral Surface Area of Cube is " << lsa;
}